cohorts/load.py

```diff
@@ -764,6 +767,43 @@ def _load_single_patient_effects(self, patient, only_nonsynonymous, filter_fn):
             patient=patient,
             filter_fn=filter_fn)
 
+    def load_kallisto(self):
+        """
+        Load a Kallisto gene expression data for a cohort
```

Comment:
technically this is loading transcript quantification. 

It might be worthwhile to annotate the gene's here as well, though it's obvious how to aggregate to gene level. Summing should be fine, but [`tximport`](https://github.com/mikelove/tximport) tries some other strategies as well.